Libraries/LibGfx/Bitmap.h

```diff
@@ -38,7 +38,8 @@
     __ENUMERATE_IMAGE_FORMAT(png, ".png") \
     __ENUMERATE_IMAGE_FORMAT(gif, ".gif") \
     __ENUMERATE_IMAGE_FORMAT(bmp, ".bmp") \
-    __ENUMERATE_IMAGE_FORMAT(ico, ".ico")
+    __ENUMERATE_IMAGE_FORMAT(ico, ".ico") \
+    __ENUMERATE_IMAGE_FORMAT(jpg, ".jpg")
```

Comment:
Since libweb matches both ".jpg" and ".jpeg", it would be a good idea to do it here too :
```suggestion
    __ENUMERATE_IMAGE_FORMAT(jpg, ".jpg") \
    __ENUMERATE_IMAGE_FORMAT(jpg, ".jpeg")
```